types/navermaps/index.d.ts

```diff
@@ -133,11 +133,11 @@ declare namespace naver.maps {
         zoomOrigin?: any;
     }
     interface MarkerOptions {
-        animation?: any;
-        map?: Map | undefined;
-        position?: any;
-        icon?: any;
-        shape?: any;
+        animation?: Animation;
+        map: Map | undefined;
```

Comment:
If this value is mandatory, the `undefined` option should probably be removed:

```suggestion
        map: Map;
```